### PR TITLE
Displays File Type

### LINKED ISSUE #18: In search results, show more info about file type
Search results currently show no info about file type, which makes it hard to tell if you're about to open a PDF, a Presentation, a PNG, or a Doc. Showing info about this will make search more useful.

See also: https://github.com/nytimes/library/issues/90

### PR DESCRIPTION
**Pull request by zenkimoto**
### Description of Change
Displays file type based on mime

### Checklist

- [x] Ran `npm run lint` and updated code style accordingly
- [ ] `npm run test` passes
- [x] PR has a description and all contributors/stakeholder are noted/cc'ed
- [ ] tests are updated and/or added to cover new code
- [x] relevant documentation is changed and/or added

### CODE REVIEW

File: custom/layouts/pages/search.ejs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    <div class="g-body">
+      <div class="g-main-content" id="g-search-page">
+        <% if (locals.results && results.length) { %>
+          <h3 class="search-kicker"><%- template('search.results.kicker', locals.q) %></h3>
+          <% results.forEach((res) => { %>
+          <div class="search-result">
+            <h3><a href="<%= res.path %>"><%= res.prettyName %></a></h3>
+            <%- include('partials/fileType', { mimeType: res.mimeType }) %>
+            <%- template('search.results.') %>
```
Comment (gunsch) [inline custom/layouts/pages/search.ejs:20]: what's this line for?

File: custom/layouts/pages/search.ejs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <% } %>
+      </div>
+
+      <%- include('partials/footer', { pageType: 'search' }) %>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/js/fontawesome.min.js" 
+              integrity="sha512-5qbIAL4qJ/FSsWfIq5Pd0qbqoZpk5NcUVeAAREV2Li4EKzyJDEGlADHhHOSSCw0tHP7z3Q4hNHJXa81P92borQ==" crossorigin="anonymous" 
+              referrerpolicy="no-referrer"></script>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/js/solid.min.js" 
+              integrity="sha512-C92U8X5fKxCN7C6A/AttDsqXQiB7gxwvg/9JCxcqR6KV+F0nvMBwL4wuQc+PwCfQGfazIe7Cm5g0VaHaoZ/BOQ==" crossorigin="anonymous" 
+              referrerpolicy="no-referrer"></script>
```
Comment (gunsch) [inline custom/layouts/pages/search.ejs:44]: are these for the font icons? maybe leave a comment indicating that

**Review by gunsch — COMMENTED** [review]
oh sweet! so they already had a custom hook for re-rendering the search page?

this looks like it'll be nice and easy for us to then add more search UI to

File: custom/layouts/pages/search.ejs (hunk 1 above)
Comment (zenkimoto) [reply custom/layouts/pages/search.ejs]: Whoops.  Thanks for noticing.  That shouldn't be there.

**Review by gunsch — APPROVED** [review]

**Pull request merged**